lib/worker-transformer.js

```diff
@@ -26,7 +26,7 @@ class WorkerCallbackTransformer extends Transformer {
     }
 
     compute(input, output) {
-        return this._callback(input, output);
+        return this._callback(input, output, input.params);
```

Comment:
Not in this PR - but when constructing the class, there should be a check that `this._callback` is a function.
```suggestion
        // TODO: Verify `this._callback` is a function.
        return this._callback(input, output, input.params);
```